src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -95,6 +95,25 @@ public CameraData()
             UpY = defaultCameraUpDirection.Y;
             UpZ = defaultCameraUpDirection.Z;
         }
+        
+        public override bool Equals(object obj)
```

Comment:
@mjkkirschner @alfarok This should fix the camera tests. I will run the self serve CI